examples/sktime/train.py

```diff
@@ -0,0 +1,68 @@
+import mlflow
+import json
+
+from sktime.datasets import load_longley
+from sktime.forecasting.model_selection import temporal_train_test_split
+from sktime.forecasting.naive import NaiveForecaster
+from sktime.performance_metrics.forecasting import (
+    mean_absolute_error,
+    mean_absolute_percentage_error,
+)
+
+import flavor
+
+ARTIFACT_PATH = "model"
+COVERAGE = [0.8, 0.9]
+
+with mlflow.start_run():
+    y, X = load_longley()
+    y_train, y_test, X_train, X_test = temporal_train_test_split(y, X)
+
+    forecaster = NaiveForecaster()
+    forecaster.fit(
+        y_train,
+        X=X_train,
+        fh=[1, 2, 3, 4],
+    )
+
+    # create prediction configuration used in pyfunc flavor
+    pyfunc_predict_conf = {
+        "predict_method": {
+            "predict": {},
+            "predict_interval": {"coverage": COVERAGE},
+        }
+    }
+
+    # assign configuration dictionary to fitted model instance
+    forecaster.pyfunc_predict_conf = pyfunc_predict_conf
```

Comment:
is direct setting the recommended way to do this? Is there not a setter? Or an argument (even a kwarg) in the fit method?

Reply:
need to check